project/plugins.sbt

```diff
@@ -20,10 +20,10 @@ addSbtPlugin("software.purpledragon" % "sbt-checkstyle-plugin" % "4.0.1")
 // sbt-checkstyle-plugin uses an old version of checkstyle. Match it to Maven's.
 // If you are changing the dependency setting for checkstyle plugin,
 // please check pom.xml in the root of the source tree too.
-libraryDependencies += "com.puppycrawl.tools" % "checkstyle" % "10.14.0"
+libraryDependencies += "com.puppycrawl.tools" % "checkstyle" % "10.15.0"
 
-// checkstyle uses guava 31.0.1-jre.
-libraryDependencies += "com.google.guava" % "guava" % "31.0.1-jre"
+// checkstyle uses guava 33.1.0-jre.
+libraryDependencies += "com.google.guava" % "guava" % "33.1.0-jre"
```

Comment:
Oh, which `checkstyle` version upgrade this?

Reply:
the version `10.15.0` use it. I have check it.
https://github.com/checkstyle/checkstyle/blob/checkstyle-10.15.0/pom.xml#L289-L292
<img width="442" alt="image" src="https://github.com/apache/spark/assets/15246973/5bf23bee-bd72-48fd-93f5-dc2359cd23ed">